bench_runner/templates/_benchmark.src.yml

```
@@ -227,9 +230,11 @@ jobs:
         run: |
           # Another benchmarking task may have created results for the same
           # commit while the above was running. This "magic" incantation means
-          # that any local results for this commit will override anything we
-          # just pulled in in that case.
-          git pull -s recursive -X ours --autostash --rebase
+          # that any local results for this commit will override anything that
+          # happened in the meantime.
+          # https://stackoverflow.com/questions/43770520/how-to-specify-default-merge-strategy-on-git-stash-pop
+          git stash
+          git cherry-pick -n -m1 -Xtheirs stash
```

Do we also need to pull here and below?
```suggestion
          git stash
          git pull -s recursive --rebase
          git cherry-pick -n -m1 -Xtheirs stash
```